app/assets/stylesheets/_cookie-banner.scss

```
@@ -0,0 +1,56 @@
+.cookies-banner {
+  background-color: $lightgrey-10;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;  
+  border-radius: 4px;
+  margin: 0 auto;
+  padding: 10px;
+  left: 20px;
+  right: 20px;
+  max-width: fit-content;
+  bottom: -400px;
+  box-shadow: $lightgrey-16 0px 0px 12px 0px;
+  z-index: 1000;
+  visibility: hidden;
+  transition: all 0.8s ease-in-out;  
+
+  .cookies-banner__msg {
```

Since we're using SCSS, we can make it a little shorter and remove some duplication with this:

```suggestion
  &__msg {
```

In SCSS/SASS files, the `&` operator is the parent of a selector, so when we nest selectors we can use this. What do you think about using it?